main.cpp

```diff
@@ -472,8 +337,136 @@ After you have MOVED your 10 UDTs, send me a DM with your pull request link.
 I will review the pseudo-code that you have written.
 */
 
+/*
+1) Electric Guitar
+5 properties:
+    1) number of strings
+    2) length of neck
+    3) number of songs played
+    4) type of pick-ups
+    5) tuning
+3 things it can do:
+    1) play a song
+    2) tune to key
+    3) change strings
+
+
+2) Gaming PC
+5 properties:
+    1) number of CPU cores
+    2) number of SSD cards
+    3) number of fans
+    4) height of PC
+    5) number of USB ports
+3 things it can do:
+    1) play games
+    2) code software
+    3) play youtube videos
+ 
+
+3) Phone
+5 properties:
+    1) Number of buttons
+    2) number of screens
+    3) number of speakers
+    4) percentage of charge
+    5) number of cameras
+3 things it can do:
+    1) make a call
+    2) text other people
+    3) set an alarm
+ 
+
+4) Audio Interface
+5 properties:
+    1) number of knobs
+    2) number of midi inputs
+    3) number of outputs
+    4) number of inputs
+    5) amount of gain
+3 things it can do:
+    1) convert analog to digital signal
+    2) turn up gain
+    3) record audio
```

Comment:
You need to write the types in `()` for all of these properties. 
```
===================
Part 1c assignment: 2 steps
===================

1) write the name of the primitive type you'll be using after each 
    property in UDTs 1-4 from Part 1b:
    pick properties that can be represented with 
    'int float double bool char std::string'.
    example: 
        Display:
            Number of Pixels (int)
            Amount of Power consumed (milliwatt-hours) (float)
            Brightness (double)
            area in cm2 (int)
            brand (std::string)
```
- [ ] revise